### ISSUE TITLE
prefer-web-first-assertion fails to detect the right variable declarator

### ISSUE THREAD

**Issue by An631** [playwright-community/eslint-plugin-playwright#286]
Whenever the expect call is used with an assigned variable, the code fails to find the appropriate variable declarator in the code.

For example:
```ts
        const locatorFoo = page.locator(".foo")
        const isBarVisible = await locatorFoo.locator(".bar").isVisible()
        expect(isBarVisible).toBe(false)
```
In the current code we are only checking for the first `VariableDeclarator` in the scope of the `isBarVisible` variable, this means that the code grabs the `locatorFoo` variable and verifies if it has a call to a non-web-first assertion like `isVisible()`. 

This causes very random and incorrect behaviors in the IDE red squiggles and in the suggested automated fixes.

I have a working change with tests to fix this already if I can get PR reviews for it.
